plugins/woocommerce-admin/client/marketing/components/CollapsibleCard/CollapsibleCard.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	Card,
+	CardBody,
+	CardDivider,
+	CardHeader,
+	CardFooter,
+	Button,
+} from '@wordpress/components';
+import { chevronUp, chevronDown } from '@wordpress/icons';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './CollapsibleCard.scss';
+
+export type CollapsibleCardProps = {
```

Comment:
Why we need to export the type?

Reply:
Think I've forgotten to remove it when I was moving things around. Removed in commit [`9955636` (#34080)](https://github.com/woocommerce/woocommerce/pull/34080/commits/9955636fe8413f7e6123a78a11c6054b475db6f1).